### PR TITLE
Stephen/docu 953

### PR DESCRIPTION
**Pull request by stepgerner**
As per this ticket https://jira.wandisco.com/browse/DOCU-953

LDMA work remains to be done, so I will not be flagging the ticket differently yet.

Phrasing of "commercial use" may be controversial, I would appreciate review on that terminology in particular.

### CODE REVIEW

File: docs/configuration-data-migrations.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## DFS NameNode properties
+
+There are a number of properties adjustable on the HDFS NameNode that can be used to prevent data migrations from stalling due to an excess of notifications, or from operating too slowly.
+
+These can all be configured within `hdfs-site.xml`.
+
+### dfs.namenode.inotify.max.events.per.rpc
+
+This value dictates the maximum number of events the namenode can send to an inotify client in a single RPC response. By default, this is set to `1000`, which should consume no more than 1MB of memory.
```
Comment (benradstone) [inline docs/configuration-data-migrations.md:17]: `namenode` should be written as such "NameNode".

Don't blame me, blame Hadoop!

File: docs/configuration-data-migrations.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+You may increase this value to allow for more events to occur simultaneously, at the cost of higher memory use and potentially lower performance, depending on the hardware your HDFS cluster is running on.
+
+We recommend setting this value to `100,000` for commercial use, provided your cluster is capable of providing at least 100MB of memory to the migration process at a given time.
```
Comment (benradstone) [inline docs/configuration-data-migrations.md:21]: Change to "for production use,"

File: docs/configuration-data-migrations.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### dfs.namenode.checkpoint.txns
+
+This value determines how many namespace transactions will trigger a checkpoint to occur, updating the filesystem metadata. 
+
+The default value for this is `1,000,000`, but we recommend increasing it to `10,000,000` for commercial use.
```
Comment (benradstone) [inline docs/configuration-data-migrations.md:27]: As above, "for production use".

File: docs/prereqs.md [hunk 4]
```diff
@@ -24,13 +24,17 @@ Ready to start? Check the prerequisites and then follow these steps to get ready
...
 
-## Machine specification
+## Machine Specification
 
 * **16 CPUs, 32 GB RAM** (minimum **4 CPUs, 16 GB RAM**)
   * If deploying LiveData Migrator on a Hadoop cluster, the host machine should be an edge node that matches at least 75-80% of the NameNode host specification.
 * **200 GB** (SSD-backed volumes) (minimum **100 GB**)
 * **2 Gbps** minimum network. Your network bandwidth must be able to cope with transferring data and ongoing changes from your source storage.
 
+## Commercial Use Configuration
```
Comment (benradstone) [inline docs/prereqs.md:34]: Change heading to "Production Use Configuration"

File: docs/prereqs.md [hunk 5]
```diff
@@ -24,13 +24,17 @@ Ready to start? Check the prerequisites and then follow these steps to get ready
...
+
+When using LiveData Migrator with moderate to larger-sized data loads, it's recommended you consider configuring [Data Migration properties](./configuration-data-migrations) to ensure smooth operation.
```
Comment (benradstone) [inline docs/prereqs.md:36]: Change "it's recommended you consider configuring" to just "configure".

Leave in "smooth operation" of course.

File: sidebars.js [hunk 6]
```diff
@@ -53,6 +53,10 @@ module.exports = {
               "type": "doc",
               "id": "configure-exclusions"
             },
+            {
+              "type": "doc",
```
Comment (benradstone) [inline sidebars.js:57]: My preference is that this goes in the *Configuration* sidebar header, above Metadata Migrations and below LiveData UI.

**Review by benradstone — CHANGES_REQUESTED** [review]
Some comments that need addressing. The rest is well written and the explanations are good.

File: docs/configuration-data-migrations.md (hunk 1 above)
Comment (stepgerner) [reply docs/configuration-data-migrations.md:17]: Hah, I missed that one - cheers!

**Pull request merged**